app/poros/category.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class Category
+  attr_reader :id, :name, :url
+
+  def initialize(data)
+    @id = data[:id]
+    @name = data[:name]
+    @url = data[:url]
```

Comment:
Nice Job pulling the info from the data hash into attributes. I don't think we need or will ever use the `url` attribute, so that one may be unnecessary.